README.md

```diff
@@ -395,10 +397,10 @@ fission:
   # mutualtls: false # if true, checkcert flag will be ignored (server cert will always be checked)
 
 policyreport:
-  enabled: "" # Name of Fission function, if not empty, Fission is enabled
-  kubeconfig: "fission" # Namespace of Fission Router, "fission" (default)
-  warning: "router" # Service of Fission Router, "router" (default)
-  maxreports: 80 # Port of service of Fission Router
+  enabled: false  #if true policyreport output is enabled
+  kubeconfig: "~/.kube/config"  # Kubeconfig file to use (only if falcoside is running outside the cluster)
+ failthreshold: 4 #events with priority above this specified integer are mapped to false and lower that those are mapped to warn (default=4)
```

Comment:
```suggestion
 failthreshold: 4 # events with a priority above this threshold are mapped to false and lower are mapped to warn (default=4)
```

the key name contains `fail` but the description talks about `false`, what's the correct semantic?